openapi.yaml

```diff
@@ -3497,8 +3500,739 @@ paths:
+      description: |-
+        Lists information about pricing and available parameters for searching and ordering products for a commercial data collection specified by the identifier `collection_id`.
+        STAC-compliant general information about the collection must be available at `GET collections/{collection_id}`.
+      tags:
+        - Commercial Data
+      security:
+        - {}
+        - Bearer: []
+      parameters:
+        - name: collection_id
+          in: path
+          description: Collection identifier
+          required: true
+          schema:
+            $ref: '#/components/schemas/collection_id'
+      responses:
+        '200':
+          description: Information about commercial data collection.
+          content:
+            application/json:
+              schema:
+                title: Commercial Collection Info
+                type: object
+                required:
+                  - id
+                  - pricing
+                  - search_parameters
+                  - order_parameters
+                properties:
+                  id:
+                    type: string
+                    description: A unique identifier for the collection, which MUST match the specified pattern.
+                    pattern: '^[\w\-\.~\/]+$'
+                  provider:
+                    type: string
+                    description: Provider of the data. 
+                  pricing: 
+                    type: object
+                    description: Information about pricing for searching and/or ordering data.
+                    required:
+                      - searching
+                      - ordering
+                    properties:
+                      searching:
+                        type: object
+                        description: Pricing for searching available products in the collection. Must be explicitly set to `null` if searching is free of charge.
+                        nullable: true
+                        properties:
+                          description:
+                            type: string
+                            description: Human-readable description of pricing.
+                          links:
+                            type: array
+                            description: |-
+                              Additional information about the pricing options.
+                      ordering:
+                        type: object
+                        description: General information about pricing for ordering products.
+                        properties:
+                          description:
+                            type: string
+                            description: Human-readable description of pricing.
+                          links:
+                            type: array
+                            description: |-
+                              Additional information about the pricing options.
+                  search_parameters:
```

Comment:
This could be handled via OGC's /queryables instead of a new interface.

Reply:
Indeed, thanks for that. And it works nicely with STAC /search.

However, the full OGC Queryables require 2 endpoints:
* `/queryables` which are available for all collections
* `{collectionId}/queryables` only for a specific one

I don't see a great benefit in having both, so I only added the second one. Do you think we should follow the spec closer and implement it fully?

I guess we could use the same for ordering parameters, what do you think? In our case they are usually the same as search parameters.

Reply:
We could document both here so that users know that they exist or at least mention the global one. But the more important one is likely the collection-specific one, indeed.